src/pykeen/ablation/ablation.py

```diff
@@ -97,7 +98,16 @@ def _set_arguments(key: str, value: str) -> None:
             hpo_config.update(d)
 
         # Add dataset to current_pipeline
-        hpo_config['dataset'] = dataset
+        if isinstance(dataset, str):
+            hpo_config['dataset'] = dataset
+        elif isinstance(dataset, dict):
+            # Training, test, and validation paths are provided
+            hpo_config['training_triples_factory'] = dataset['training_triples_factory']
```

Comment:
should we remove the suffix "triples_factory" since this can also be a string now?

Reply:
Good idea!
I made the changes in 4a892c91fe8d3723fceb33e9fed7d42e2171be00. Please note that I updated the `pipeline` accordingly.